crew

```diff
@@ -644,13 +644,13 @@ def prepare_package (destdir)
     system 'tail -n +2 dlistcut > dlist'
 
     # remove temporary files
-    "FileUtils.rm_rf ['dlistcut', '../dlist', '../filelist']#{@fileutils_verbose}"
+    FileUtils.rm_rf "['dlistcut', '../dlist', '../filelist']#{@fileutils_verbose}"
   end
 end
 
 def strip_find_files (find_cmd, strip_option = "")
   # check whether crew should strip
-  return if CREW_NOT_STRIP || ENV['CREW_NOT_STRIP'] || !File.exist?("#{CREW_PREFIX}/bin/llvm-strip")
+  return if CREW_NOT_STRIP || ENV['CREW_NOT_STRIP'] || !File.exist?("#{CREW_PREFIX}/bin/strip")
```

Comment:
This overwrites @satmandu's previous commit.

Reply:
Done